components/footer/index.tsx

```diff
@@ -0,0 +1,250 @@
+      },
+      {
+        label: "Staking",
+        href: "#",
+      },
+      {
+        label: "Pools",
+        href: "#",
+      },
+      {
+        label: "Apps",
+        href: "#",
+      },
+    ],
+  },
+  {
+    title: "Token",
+    links: [
+      {
+        label: "OSMO",
+        href: "#",
+      },
+      {
+        label: "CoinGecko",
+        href: "#",
+      },
+      {
+        label: "CoinMarketCap",
+        href: "#",
+      },
+      {
+        label: "Governance",
+        href: "#",
+      },
+      {
+        label: "Vote",
+        href: "#",
+      },
+    ],
+  },
+  {
+    title: "Community",
+    links: [
+      {
+        label: "X (Twitter)",
+        href: "#",
+      },
+      {
+        label: "Telegram",
+        href: "#",
+      },
+      {
+        label: "Discord",
+        href: "#",
+      },
+      {
+        label: "Forum",
+        href: "#",
+      },
+    ],
+  },
+  {
+    title: "Developers",
+    links: [
+      {
+        label: "Docs",
+        href: "#",
+      },
+      {
+        label: "GitHub",
+        href: "#",
+      },
+      {
+        label: "Grants",
+        href: "#",
+      },
+    ],
+  },
+  {
+    title: "Team",
+    links: [
+      {
+        label: "Contact",
+        href: "#",
+      },
+      {
+        label: "Blog",
+        href: "#",
+      },
+      {
+        label: "Careers",
+        href: "#",
+        isNew: true,
+      },
+    ],
+  },
+];
+
+const socials = [
+  {
+    name: "X",
+    icon: "/assets/icons/x.svg",
+    href: "#",
+  },
+  {
+    name: "Reddit",
+    icon: "/assets/icons/reddit.svg",
+    href: "#",
+  },
+  {
+    name: "Telegram",
+    icon: "/assets/icons/telegram.svg",
+    href: "#",
+  },
+];
+
+export default function Footer() {
+  {
+    /**px-4 on mobile to compensate the layout px-4 */
+  }
+  return (
+    <footer className="mb-7 self-stretch px-4 sm:mb-8 md:mb-17.5 lg:mb-16 xl:mb-[38px] 2xl:mb-[54px] 2xl:px-6">
+      <div className="flex flex-col gap-2 self-stretch lg:gap-4 2xl:gap-6">
+        {/**divider */}
+        <div className="flex items-center gap-2.5 py-6">
+          <div className="h-[1px] flex-1 bg-osmoverse-750" />
+          <Image
+            src={"/assets/icons/star-divider.svg"}
+            alt="Divider"
+            width={21}
+            height={21}
+          />
+          <div className="h-[1px] flex-1 bg-osmoverse-750" />
+        </div>
+        <div className="relative grid grid-cols-2 gap-x-8 gap-y-6 md:grid-cols-3 lg:max-w-[764px] lg:grid-cols-5 lg:gap-x-4 2xl:max-w-[1020px] 2xl:gap-x-20">
+          {sections.map(({ links, title }) => (
+            <div key={title} className="flex flex-col gap-2.5">
+              <span className="whitespace-nowrap text-sm font-light leading-[21px] text-[#F9F8F799]">
+                {title}
+              </span>
+              <div className="flex flex-col gap-0.5">
+                {links.map(({ label, href, isNew }) => (
+                  <Link
```

Comment:
For the refactor we can move this component into a reusable one, an equal behavior also happens in the header